packages/snaps-sdk/src/jsx/components/form/FileInput.ts

```diff
@@ -0,0 +1,53 @@
+import { createSnapComponent } from '../../component';
+
+/**
+ * The props of the {@link FileInput} component.
+ *
+ * @property name - The name of the file input field. This is used to identify
+ * the file input field in the form data.
+ * @property label - The label of the file input field.
+ * @property accept - The file types that the file input field accepts. If not
+ * specified, the file input field accepts all file types.
+ * @property compact - Whether the file input field is compact. Default is
+ * `false`.
+ */
+export type FileInputProps = {
+  name: string;
+  accept?: string[] | undefined;
+  compact?: boolean | undefined;
+};
+
+const TYPE = 'FileInput';
+
+/**
+ * A file input component, which is used to create a file input field. This
+ * component can only be used as a child of the {@link Field} component.
+ *
+ * The total size of the files that can be uploaded may not exceed 64 MB.
+ *
+ * @param props - The props of the component.
+ * @param props.name - The name of the file input field. This is used to
+ * identify the file input field in the form data.
+ * @param props.label - The label of the file input field.
+ * @param props.accept - The file types that the file input field accepts. If
```

Comment:
Could we elaborate on the format here, is it MIME types?

Reply:
It's extensions, mime types, and certain wildcard specifiers: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept

Will add this link to the docs.